### ISSUE TITLE
Approval – building block: Inputs and outputs

### ISSUE THREAD

**Issue by mica000** [leather-wallet/extension#4898]
![image](https://github.com/leather-wallet/extension/assets/119523541/4dc3167a-1adb-46d0-87e2-3046ef69f1fa)

[Notion 🔗](https://www.notion.so/trustmachines/7beb8479a0f54db9abc2186fd0c4317a?v=1e50d41e5640407281e28e4faaacf221&p=f18b68712c8347bc81ccac39b45b1410&pm=s)

[Figma 🔗](https://www.figma.com/file/FlmF7ropcijOb7iVSDqNn0/%F0%9F%9F%A7-Web-App-Explorations-2024-H1?type=design&node-id=820-194517&mode=design&t=wOD7RuWDF7QrYEvo-4)

**Comment by kyranjamie** [comment]
The UI LGTM, but I have to reiterate copy feedback regarding use of the word "guaranteed". 

Sighash refers to the mutability only. Phrasing this sentence differently highlights the problem:

> By signing this UTXO it is guaranteed

Guaranteed _to what?_ To be spent? To not change? It's ambiguous.

Consider use case of apps that preemptively sign a handful of PSBT to handle different scenarios. Think Gamma does this with Dutch auctions? You might sign 4 PSBTs, however only one is ever signed by the other party. Currently, Leather might say they're all "guaranteed" to happen, which isn't accurate.

**Comment by mica000** [comment]
@kyranjamie thanks for the feedback! Any suggestions? 
What do you think of "this UTXO is immutable" and "this UTXO can change after signing"?

**Comment by kyranjamie** [comment]
Yes, sorry for not suggesting alternatives. Recognise this is hard to explain. Some ideas:


> _This input/output is (im)mutable_

Technically accurate

> _This input/output can be modified_

Avoids immutable which might be a complicated word we choose to avoid

> _The amount of bitcoin in this output can be changed by another party_

Focuses on the primary outcome set by the sig hash

**Comment by mica000** [comment]
@kyranjamie Giving we would have different scenarios, would this summarise well?

> The amount of {bitcoin or ordinal} in this {input or output} {can or can't} be changed by another party
please, correct me if Im wrong?

>Both the amount and recipient of this output can be modified by the app
It's not only the amount? Are you suggesting we integrate this information on the tooltip as well?

cc @markmhendrickson for copy feedback as well
